README.md

```diff
@@ -114,6 +114,41 @@ In the other OSs not listed above, the pure-java SQLite is used. (Applies to ver
 
 If you want to use the native library for your OS, [build the source from scratch](./CONTRIBUTING.md).
 
+## GraalVM native-image support
+
+Sqlite JDBC supports [GraalVM native-image](https://www.graalvm.org/native-image/) out of the box starting from version 3.40.1.0.
```

Comment:
I'm not entirely sure if this is the right version to quote.

Reply:
It should be fine.